pkgs/top-level/all-packages.nix

```diff
@@ -14653,8 +14653,18 @@ in
 
   rpcbind = callPackage ../servers/rpcbind { };
 
-  mariadb = callPackage ../servers/sql/mariadb {
+  libmysql = mariadb-connector-c;
+  mariadb-connector-c = mariadb-connector-c_2_3;
+  mariadb-connector-c_2_3 = callPackage ../servers/sql/mariadb/connector-c/2_3.nix { };
+  mariadb-connector-c_3_0 = callPackage ../servers/sql/mariadb/connector-c/3_0.nix { };
+  mariadb-connector-c_3_1 = callPackage ../servers/sql/mariadb/connector-c/3_1.nix { };
```

Comment:
Any reason to have 3.0, when we already have 3.1?

Reply:
Could we also just drop 2.3? Archlinux does this as well.

Reply:
I think first leave this variant. In the next stage change to 3.0 / 3.1

Reply:
For which reasons?

Reply:
Then it will be easier to check package builds via nix-review

Reply:
You can only build a subset in `nix-review` by using the `-p` option.

Reply:
with
`nix-review pr -p mariadb-connector-c -p libmysql 65222`
checked only mariadb-connector-c libmysql

Reply:
@Izorkin what would you expect instead?

Reply:
@Mic92 automatic build of packages that depend on libmysql